src/edits.rs

```diff
@@ -228,14 +246,18 @@ where
                     },
                     minus_section,
                 ));
-                annotated_plus_line.push((
-                    if coalesce_space_with_previous {
-                        plus_op_prev
-                    } else {
-                        noop_insertion
-                    },
-                    plus_section(n, &mut y_offset),
-                ));
+                let op = if coalesce_space_with_previous {
+                    plus_op_prev
+                } else {
+                    noop_insertion
+                };
+                let line = plus_section(n, &mut y_offset);
```

Comment:
It looks like this variable should be named `plus_section` for consistency with `minus_section` and `plus_section` elsewhere in this function.